projects/plugins/boost/app/assets/src/js/pages/getting-started/getting-started.module.scss

```diff
@@ -0,0 +1,9 @@
+.pricing-table {
+	isolation: isolate;
+
+	h2 {
+		font-size: 36px !important;
+		font-weight: 700 !important;
+		line-height: 40px !important;
+	}
```

Comment:
Why is this so `!important`

Reply:
I guess I found a way without it - thanks 😅